bigml/modelfields.py

```diff
@@ -138,6 +139,77 @@ def check_model_fields(model):
     return False
 
 
+def get_datetime_subfields(fields):
+    """ From a dictionary of fields, returns another dictionary
+    with the subfields from each datetime field
+
+    """
+    subfields = {}
+    for fid, finfo in fields.items():
+        if (finfo.get('parent_optype', False) == 'datetime'):
+            parent_name = ".".join(finfo["name"].split(".")[:-1])
```

Comment:
this is not very robust, because users can change the field names. You could use the "datatype" property, right?